code/controllers/subsystem/ore_generation.dm

```diff
@@ -57,6 +36,20 @@ SUBSYSTEM_DEF(ore_generation)
 				if(stallbreaker >= length(possible_vents))
 					return SS_INIT_SUCCESS //We've done all we can here.
 				continue
+
+	/// Handles roundstart logging
+	logger.Log(
+		LOG_CATEGORY_CAVE_GENERATION,
+		"Ore Generation spawned the following ores based on vent proximity: 1:[GLOB.post_ore_random["1"]], 2:[GLOB.post_ore_random["2"]], 3:[GLOB.post_ore_random["3"]], 4:[GLOB.post_ore_random["4"]], 5:[GLOB.post_ore_random["5"]]",
+	)
+	logger.Log(
+		LOG_CATEGORY_CAVE_GENERATION,
+		"Ore Generation spawned the following ores randomly: 1:[GLOB.post_ore_manual["1"]], 2:[GLOB.post_ore_manual["2"]], 3:[GLOB.post_ore_manual["3"]], 4:[GLOB.post_ore_manual["4"]], 5:[GLOB.post_ore_manual["5"]]",
+	)
+	logger.Log(
+		LOG_CATEGORY_CAVE_GENERATION,
+		"Ore Generation spawned the following vent sizes: large:[GLOB.ore_vent_sizes[LARGE_VENT_TYPE]], medium:[GLOB.ore_vent_sizes[MEDIUM_VENT_TYPE]], small:[GLOB.post_ore_manual[SMALL_VENT_TYPE]]",
```

Comment:
```suggestion
		"Ore Generation spawned the following vent sizes: large:[LAZYACCESS(GLOB.ore_vent_sizes, LARGE_VENT_TYPE)], medium:[LAZYACCESS(GLOB.ore_vent_sizes, MEDIUM_VENT_TYPE)], small:[LAZYACCESS(GLOB.ore_vent_sizes, SMALL_VENT_TYPE)]",
```

Your whole subsystem & logging might be crashing because of bad list index runtime. Byond doesn't parse string defines properly when accessing lists so you have to do this

Also you are accessing `SMALL_VENT_SIZE` from `post_ore_manual` list ? That entry doesn't exist leading to even more runtimes

Reply:
That the thing, It hasn't been runtiming, just not logging anything. If it was runtiming in theory it should be showing up immediately on compile, or, at least creating an entry to the .log file where the actual `GLOB.post_ore_manuel[xyz]` should just not be providing a result.

Reply:
> That the thing, It hasn't been runtiming

Sadly VS code debugger stopped working after the 515 update more than 3 months ago so you would not even have noticed it. Try going to the debugger tab & see view runtimes instead.

Anyway now since you have applied the fix let me know if logging works now

Reply:
You can resolve this and see my below reviews